webots/controllers/RL_Supervisor/RL_Supervisor.py

```diff
@@ -0,0 +1,144 @@
+"""Supervisor controller.
+    Details: https://github.com/cyberbotics/webots/blob/master/docs/guide/supervisor-programming.md
+"""
+import sys
+from Serial_webots import SerialWebots
+from SerialMuxProt import SerialMuxProt
+from controller import Supervisor
+import struct
+
+# The PROTO DEF name must be given!
+ROBOT_NAME = "ROBOT"
+
+# The supervisor receiver name.
+SUPERVISOR_RX_NAME = "serialComRx"
+
+# The supervisor Emitter name.
+SUPERVISOR_TX_NAME = "serialComTx"
+
+# Create the Supervisor instance.
+supervisor = Supervisor()
+
+# Get the time step of the current world.
+timestep = int(supervisor.getBasicTimeStep())
+
+# Get the Supervisor Receiver Device
+supervisor_com_rx = supervisor.getDevice(SUPERVISOR_RX_NAME)
+
+# Get the Supervisor Emitter device 
+supervisor_com_Tx = supervisor.getDevice(SUPERVISOR_TX_NAME)
```

Comment:
Adhere PEP 8 style guide.

Imports:
Imports should be grouped into standard library imports, related third-party imports, and local application/library-specific imports. Each group should be separated by a blank line.

Constants:
Constants should be defined in uppercase with words separated by underscores.

Variable Naming:
Variable names should be in snake_case.

Global Variables:
Avoid using global variables. If necessary, encapsulate them within a class or pass them as parameters.
String Formatting:

Indentation:
Ensure consistent indentation. Use 4 spaces per indentation level.

Comments:
Comments should be complete sentences and start with a capital letter.

Reply:
Ensure you have PyLint Extension installed to provide support while adhering to the PEP8 style guide.
Style will be also checked by the CI (#136).